release_notes/ocm.py

```diff
@@ -380,3 +381,54 @@ def release_note_for_ocm_component(component: ocm.Component) -> str:
         component_release_notes += category_markdown
 
     return component_release_notes
+
+
+def iter_release_notes_docs(
```

Comment:
there is already `release_notes_for_subcomponents`. I think it will make more sense to re-use this function, and filter the returned ReleaseNotesDoc-iterable (recursion-depth is already prepared as a non-exposed parameter, there).